.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "ocluster"]
+	path = ocluster
+	url = https://github.com/patricoferris/ocluster
+	branch = macos-v2
```

Comment:
does this really need the ocluster fork? I’m only using the `ocluster-api` library which the macos doesn’t change afaik

Reply:
Nope quite right, an artefact of the setup I had 😅